src/data/set/basic.lean

```diff
@@ -553,6 +562,17 @@ by finish [ext_iff]
 @[simp] theorem compl_univ : -(univ : set α) = ∅ :=
 by finish [ext_iff]
 
+lemma compl_empty_iff {s : set α} : -s = ∅ ↔ s = univ :=
+by { split, intro h, rw [←compl_compl s, h, compl_empty], intro h, rw [h, compl_univ] }
+
+lemma compl_univ_iff {s : set α} : -s = ∅ ↔ s = univ :=
```

Comment:
This one is identical to the previous one. Which is not what you intended... I guess

Reply:
He probably meant `-s = empty iff s = univ`. I think both of these should be @[simp]

Reply:
The name suggests: `-s = univ ↔ s = ∅`

Reply:
oops, copy-paste without actually changing the statement. Yes, Johannes wrote the statement I had in mind.